pandas/tests/extension/test_datetime.py

```diff
@@ -162,6 +162,20 @@ def _compare_other(self, s, data, op_name, other):
         # with (some) integers, depending on the value.
         pass
 
+    def test_compare_with_Categorical(self):
```

Comment:
We shouldn't add additional tests to this file (some other files in this directory have a note about that, just opened a PR to add that to this file as well -> https://github.com/pandas-dev/pandas/pull/39003)

@ftrihardjo can you move the test to eg `/tests/arrays/test_datetimelike.py` ? (not fully sure what is the best place to put those nowadays) There also already seems to be a `test_compare_categorical_dtype` test in that file, so you can put it close to that one (or see to what extent it overlaps)